test/ClangImporter/overlay.swift

```diff
@@ -15,7 +15,7 @@ import Redeclaration
 let encoding: UInt = NSUTF8StringEncoding
 
 let viaTypedef: Redeclaration.NSPoint = AppKit.NSPoint(x: 0, y: 0)
-Redeclaration.NSStringToNSString(AppKit.NSStringToNSString("abc")) // expected-warning {{result of call to 'NSStringToNSString' is unused}}
+let _ = Redeclaration.NSStringToNSString(AppKit.NSStringToNSString("abc"))
```

Comment:
Does this declaration lose warn_unused_result? It looks like both versions of the declaration have it, and I assume we want to test that it gets applied.

Reply:
My bad, I'll bring this back.